Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -2126,7 +2126,7 @@ static void setDynamicRangeLimit(CALayer* layer, PlatformDynamicRangeLimit platf
 void WebViewImpl::windowWillBeginSheet()
 {
 #if ENABLE(POINTER_LOCK)
-    m_page->requestPointerUnlock();
+    m_page->requestPointerUnlock(nullptr);
```

Comment:
This looks like it will crash.  Give it a lambda that does the equivalent of WebPage::DidAcquirePointerLock if successful, something else if unsuccessful.  Same with all the places you pass nullptr.